pkg/rpm/database.go

```diff
@@ -41,6 +41,12 @@ const (
 	// databaseDir is the directory where the RPM database is expected to be in
 	// the container filesystem.
 	databaseDir = "var/lib/rpm"
+
+	// rhcosDbDir is an alternative to databaseDir, as RHCOS saves its RPM DB in
+	// a different path that symlinks to var/lib/rpm.
+	// Since PR #1014, Scanner code follows symlinks , but usr/share/rpm will get filtered out
+	// as it is not a well known database dir unless defined here.
+	rhcosDbDir = "usr/share/rpm"
```

Comment:
nit: `rhcosDBDir`

Reply:
Another option is to do something like:

```
databaseDirs = []string{"var/lib/rpm", "usr/share/rpm"}
```

and have nested for loops through `darabaseDirs` and `databaseFiles`

Reply:
I chose a separate var to improve understandabiliy, combined with the detailed code comment explaining the var.
That said, I'm open to changing it to a nested loop if that fits the rest of the codebase better.
WDYT?